cli/tauri.js/bin/tauri.js

```diff
@@ -32,24 +34,27 @@ const tauri = function (command) {
     command === '--help' ||
     command === 'help'
   ) {
-    console.log(`
-    Description
-      This is the Tauri CLI.
-    Usage
-      $ tauri ${cmds.join('|')}
-    Options
-      --help, -h     Displays this message
-      --version, -v  Displays the Tauri CLI version
-    `)
+    
+    console.log(chalk.cyan(figlet.textSync('Tauri')));
+    console.log(chalk.green('Description'))
+    console.log('This is the Tauri CLI');
+    console.log(chalk.magenta('Usage'));
+    console.log('$ tauri ${cmds.join('|')}')
+    console.log(chalk.blue('Options'))
+    console.log('--help, -h     Displays this message \n  --version, -v  Displays the Tauri CLI version');
     process.exit(0)
     // eslint-disable-next-line no-unreachable
     return false // do this for node consumers and tests
   }
 
   if (command === '-v' || command === '--version') {
-    console.log(require('../package.json').version)
+    console.log(chalk.green(`${package.version}`));
+    // also added colors to beautify the cli
     return false // do this for node consumers and tests
   }
+  if (command ==='-no-update-notifier' || command === '--no-update-notifier'){
```

Comment:
I don't think this check will work as it won't detect `tauri build --no-update-notifier`, right? You should check the `process.argv`, and you only need to look for the `--` variant, not `-`.